examples/tutorial/README.rst

```diff
@@ -31,6 +31,11 @@ Or on Windows cmd::
     py -3 -m venv venv
     venv\Scripts\activate.bat
 
+Or on Windows PowerShell::
+
+    python -m venv venv
```

Comment:
Still need to use `py -3` instead of `python` here. Does `venv\Scripts\activate` work in CMD as well? If so we can consolidate the two for this step.